### PR TITLE
Add action + output

### PR DESCRIPTION
**Pull request by backnight**
+ added ses:SendEmail action
+ added secret var output

### CODE REVIEW

File: outputs.tf [hunk 1]
```diff
@@ -13,6 +13,12 @@ output "this_access_key" {
   description = "IAM Access Key of the created user, used as the STMP user name"
 }
 
+output "this_access_secret" {
+  value       = element(concat(aws_iam_access_key.this.*.secret, [""]), 0)
+  description = "IAM Access Secret of the created user"
```
Comment (danvaida) [inline outputs.tf:18]: Hey @backnight, thanks for your contribution!
As the name of this module implies, its sole purpose is to create IAM users for relaying e-mails through SES.
As such, the secret access key is not required. Only thing required is the access key and the smtp password (see below).

**Comment by danvaida** [comment]
Please see my comment [here](https://github.com/Flaconi/terraform-aws-iam-ses-user/pull/7/files/1dad616a506c44687878a7b63ea03d7e480d5998#r469756422). For creating IAM users, I'd like to point you to this other module: https://github.com/Flaconi/terraform-aws-iam-user

File: outputs.tf (hunk 1 above)
Comment (backnight) [reply outputs.tf:18]: Hello @danvaida, please have a look at this stackoverflow answer: https://stackoverflow.com/questions/14296999/status-code-403-signaturedoesnotmatch-when-i-am-using-amazon-ses/14297000#14297000

When using Amazon SES API to send email (and not the smtp endpoint), you need to provide the secret access key.
For example, node-ses (https://www.npmjs.com/package/node-ses) uses the SES API.

Comment (danvaida) [reply outputs.tf:18]: For interacting with SES via its API, we'd most likely use [this](https://github.com/Flaconi/terraform-aws-iam-user) TF module.

However, I am inclined to believe that your use case is also legitimate. Would you be willing to introduce this feature by using a toggle like `enable_api_access`? When setting that boolean to `true`, the IAM policy would be extended with the additional action and then output would contain the secret access key.

Comment (backnight) [reply outputs.tf:18]: Hello, sorry for the delayed answer, just came back from holidays.
I'm not a terraform expert so I dont know if I can do this. I will try during the next WE.

**Pull request closed without merging**